src/nvim/ex_cmds2.c

```diff
@@ -868,7 +868,7 @@ void set_context_in_profile_cmd(expand_T *xp, char_u *arg)
     return;
   }
 
-  // TODO: expand function names after "func
+  // TODO: expand function names after "func // NOLINT(readability/todo)
```

Comment:
@phanimahesh Yes, annotate. Use nick: I see `TODO(tarruda)` in some files (e.g. os/input.c:88).